spec/unit/server_spec.rb

```diff
@@ -60,28 +60,49 @@ def app
   end
 
   describe "show" do
-    describe "GET /:id" do
-      let(:description_d) {
-        create_test_description(
-          name: "description_d",
-          store_on_disk: true,
-          scopes: ["os"],
-          format_version: 7
-        )
-      }
+    context "description with incompatible format version" do
+      describe "GET /:id" do
+        let(:description_d) {
```

Comment:
I would use speaking names like "older_format_version" and for the other "unknown_format_version" or "newer_format_version"